README.md

```diff
@@ -340,6 +340,31 @@ Further Information:
 * See [Configuration Properties](#properties) on how to enable basic authentication for REST, create a default user, or disable the redirect.
 * Enabling the REST API or the Webapps impacts the startup time. Depending on your hardware it increases by around 500-1000 milliseconds.
 
+### Advanced Webapps Configuration
+It is possible to configure the CsrfPreventionFilter and the HttpHeaderSecurityFilter with the following properties.
```

Comment:
* I think it would be better to not reference the implementation classes here, i.e. "The security of the Webapps can be configured with the following properties:"
* Let's collapse the table by default.